kubernetes/linux/setup.sh

```diff
@@ -40,9 +40,9 @@ gem uninstall time --version 0.2.0
 gem uninstall uri --version 0.11.0
 
 if [ "${ARCH}" != "arm64" ]; then
-    wget "https://github.com/microsoft/Docker-Provider/releases/download/official%2Fmdsd%2F1.26.1/azure-mdsd-1.26.1-build.master.97.x86_64.rpm" -O azure-mdsd.rpm
+    wget "https://github.com/microsoft/Docker-Provider/releases/download/official%2Fmdsd%2F1.27.4/azure-mdsd-1.27.4-build.master.140.x86_64.rpm" -O azure-mdsd.rpm
```

Comment:
this is the same change with PR: update mdsd to 1.27.4  right?  do you want to combine the PR